corehq/apps/api/resources/meta.py

```diff
@@ -1,27 +1,55 @@
 from django.conf import settings
 
 from tastypie.authorization import ReadOnlyAuthorization
-from tastypie.throttle import CacheDBThrottle
+from tastypie.throttle import BaseThrottle
 
 from corehq.apps.api.resources.auth import LoginAndDomainAuthentication
 from corehq.apps.api.serializers import CustomXMLSerializer
+from corehq.project_limits.rate_limiter import (
+    PerUserRateDefinition,
+    RateDefinition,
+    RateLimiter,
+    get_dynamic_rate_definition,
+)
+from corehq.project_limits.shortcuts import get_standard_ratio_rate_definition
 from corehq.toggles import API_THROTTLE_WHITELIST
 
 
+
+api_rate_limiter = RateLimiter(
+    feature_key='api',
+    get_rate_limits=lambda domain: _get_per_user_api_rate_definition(domain)
```

Comment:
instead of using a lambda here you could use the same pattern as elsewhere: `get_rate_limits=PerUserRateDefinition(...).get_rate_limits`

https://github.com/dimagi/commcare-hq/blob/6afc542e57ac7a9986c7e3b09c5b7615217cac59/corehq/apps/ota/rate_limiter.py#L15-L27

This has the added advantage of not having to re-created the objects every time